storage_scrubber/src/metadata_stream.rs

```diff
@@ -114,9 +114,10 @@ pub async fn stream_tenant_timelines<'a>(
                 prefix.get_path().as_str().strip_prefix(prefix_str)
             })
             .map(|entry_id_str| {
-                entry_id_str
+                let first_part = entry_id_str.split('/').next().unwrap();
```

Comment:
does `/` always exist in the entry ID for all backends? Otherwise it would panic and we should use `strip` instead?

Reply:
it doesn't exist for AWS. We could also use `trim_end_matches` here, shrug.